### PR TITLE
Glitch

### PR DESCRIPTION
**Pull request by komasystem**

### CODE REVIEW

**Comment by AmirjohnSanhinov** [comment]
This pull request is either OK or not OK.

**Comment by komasystem** [comment]
Why?

File: task_6_loop_addition.js [hunk 1]
```diff
@@ -1,96 +1,96 @@
...
-       	a++;
-       	b--;
-  	}
-   	return a;
-}
-
-
-
-button.addEventListener("click", function() {
-		if ( (input1.value == "") || (input2.value == "") ) {
```
Comment (AmirjohnSanhinov) [inline task_6_loop_addition.js:63]: I don't like this line, for example.

**Comment by BabajanovKomron** [comment]
.

File: task_6_loop_addition.js [hunk 2]
```diff
@@ -1,96 +1,96 @@
...
-			result.innerHTML = "one or both operands are empty";
```
Comment (komasystem) [inline task_6_loop_addition.js:64]: broohaha

File: task_6_loop_addition.js [hunk 3]
```diff
@@ -1,96 +1,96 @@
...
-		
-		var ar1 = Number(input1.value);
-		var ar2 = Number(input2.value);
-		if ( (ar1 == ar1) && (ar2 == ar2) ) { // NaN is never equal to itself. So, (ar1 == ar1) is the same as (ar1 != NaN)
-			result.innerHTML += loopMultiplication(ar1, ar2);
-			result.innerHTML += "<br/>"; // this element is a new line
-			result.innerHTML += loopExponentiation(ar1, ar2);
-			result.innerHTML += "<br/>"; // this element is a new line
-			result.innerHTML += loopAddition(ar1, ar2);
-			result.innerHTML += "<br><br>";
```
Comment (AmirjohnSanhinov) [inline task_6_loop_addition.js:76]: and here I would add `<hr>` instead of `<br>`

**Review by AmirjohnSanhinov — APPROVED** [review]
I have left a comment

**Pull request merged**